Server_Side_TLS.mediawiki

```diff
@@ -35,170 +35,167 @@ The ordering of a ciphersuite is very important because it decides which algorit
 == <span style="color:green;">'''Modern'''</span> compatibility ==
 For services that don't need backward compatibility, the parameters below provide a higher level of security. This configuration is compatible with Firefox 27, Chrome 30, IE 11 on Windows 7, Edge, Opera 17, Safari 9, Android 5.0, and Java 8.
 
-* Ciphersuites: '''ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256'''
-* Versions: '''TLSv1.2'''
-* TLS curves: '''prime256v1, secp384r1, secp521r1'''
-* Certificate type: '''ECDSA'''
-* Certificate curve: '''prime256v1, secp384r1, secp521r1'''
+* Ciphersuites: '''ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256'''
```

Comment:
are you sure that your OpenSSL 1.1.0 is compiled with AES-NI support and is running on CPU that supports it? I'm getting over 5GiB/s for aes-128-gcm on i7-6600U CPU @ 2.60GHz:
```
type             16 bytes     64 bytes    256 bytes   1024 bytes   8192 bytes  16384 bytes
aes-128-gcm     496908.87k  1272636.25k  2384261.14k  3661835.68k  5025197.46k  5093599.29k
aes-256-gcm     506347.22k  1202380.63k  2100516.94k  3170329.34k  3496019.56k  3559158.24k
chacha20        304854.80k   550133.96k  1324290.14k  2736866.65k  2819459.00k  2773707.09k
chacha20-poly1305   235724.31k   510298.69k   959805.56k  1741361.66k  1736591.29k  1791584.90k
```